src/LibSassBuilder/Program.cs

```diff
@@ -93,11 +93,13 @@ async Task CompileFilesAsync(IEnumerable<string> sassFiles)
 				var result = SassCompiler.CompileFile(file, options: Options.SassCompilationOptions);
 
 				var newFile = fileInfo.FullName.Replace(fileInfo.Extension, ".css");
+				var newMapFile = fileInfo.FullName.Replace(fileInfo.Extension, ".css.map");
 
-				if (File.Exists(newFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newFile)).ReplaceLineEndings())
-					continue;
+				if (!(File.Exists(newFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newFile)).ReplaceLineEndings()))
+					await File.WriteAllTextAsync(newFile, result.CompiledContent);
 
-				await File.WriteAllTextAsync(newFile, result.CompiledContent);
+				if (!(string.IsNullOrEmpty(result.SourceMap) || (File.Exists(newMapFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newMapFile)).ReplaceLineEndings())))
```

Comment:
I think we should update the file even if the string is empty - perhaps they did that on purpose?

Reply:
WriteAllTextAsync checks contents for null and empty. So we can skip that and have the same result.
But maybe we should delete the file instead and no let a potentially wrong map file lie around?